Valet/VALValet.m

```diff
@@ -608,8 +608,9 @@ - (nonnull NSString *)secServiceIdentifier;
 - (nullable NSString *)_sharedAccessGroupPrefix;
 {
     NSDictionary *query = @{ (__bridge NSString *)kSecClass : (__bridge NSString *)kSecClassGenericPassword,
-                             (__bridge id)kSecAttrAccount : @"SharedAccessGroupPrefixPlaceholder",
```

Comment:
Changed the name so that we don't get bitten by an old entry with incorrect permissions